chapter_linear-networks/linear-regression-gluon_vn.md

```diff
@@ -268,29 +268,34 @@ Once we start working with much more complex models, Gluon's advantages will gro
 However, once we have all the basic pieces in place, the training loop itself is strikingly similar to what we did when implementing everything from scratch.
 -->
 
-*dịch đoạn phía trên*
+Bạn có thể thấy rằng việc thể hiện mô hình của chúng ta thông qua Gluon đòi hỏi một vài dòng lệnh tương đối.
+Chúng ta cần phải phân bổ các tham số một cách riêng lẻ, xác định hàm số mất mát của chúng ta , hay hạ gradient ngẫu nhiên.
+Một khi chúng ta làm với nhiều mô hình rất phức tạp , những lợi ích của Gluon sẽ tăng lên nhiều.
+Tuy nhiên, một khi chúng ta có tất cả các mảnh cơ bản ở đúng chỗ, vòng huấn luyện của chính bản thân rất giống với những gì chúng ta đã thực hiện tự lúc đầu.
 
 <!--
 To refresh your memory: for some number of epochs, we will make a complete pass over the dataset (train_data), iteratively grabbing one minibatch of inputs and the corresponding ground-truth labels.
 For each minibatch, we go through the following ritual:
 -->
 
-*dịch đoạn phía trên*
+Để làm mới lại bộ nhớ của bạn: đối với một số epoch, chúng ta sẽ thực hiện qua tập dữ liệu (train_data), cứ lấy lặp đi lặp lại một minibatch của dữ liệu dầu vào và gán các nhãn gốc tương ứng.
```

Comment:
```suggestion
Để làm mới lại bộ nhớ của bạn: đối với một số epoch, chúng ta sẽ thực hiện qua tập dữ liệu (train_data), cứ lấy lặp đi lặp lại một minibatch của dữ liệu đầu vào và gán các nhãn gốc tương ứng.
```

Reply:
"To refresh your memory" dịch "Để làm mới lại bộ nhớ của bạn:" nghe hơi word-to-word nhỉ, không biết có cách dịch nào hay hơn không ạ?

Reply:
@honghanhh mình nghĩ chỗ "To refresh your memory" nên dịch là "Để giúp bạn có thể nhớ thì"

Reply:
```suggestion
Để nhắc lại: cho một vài epoch, chúng ta sẽ duyêt qua toàn bộ tập dữ liệu (train_data) bằng cách lấy theo từng minibatch của dữ liệu đầu vào cộng với các nhãn gốc tương ứng.
```